### ISSUE TITLE
Shortcuts

### ISSUE THREAD

**Issue by MohabElgammal** [surveyjs/editor#79]
Dear Andrew,
I want from you To add answers of questions of  using keyboard shortcuts 
Example:your gender? Male(m),Female(f)
Thank you,
Mohab

**Comment by andrewtelnov** [comment]
@MohabElgammal It will be relatively easy to add accessKey into question title.
There is another story about choices property. At the current moment it is an array of ItemValue objects, with two properties: value and text. It will require a lot of work to implement it. Unless other people ask about this functionality, I would postpone it. 
Do you know any survey that has this functionality?

Thank you,
Andrew

**Comment by MohabElgammal** [comment]
Thank you Andrew for your quick response
yes,it is available in most of the surveys run throw agent to make it easy
the only thing required is to add field in the answer propriety which we can call it for example: stroke key and when the user define the answer is male and the stroke key is (m),i will use this field when i puplish the survey for the agent to be use  the keyboard with the shortcut key (m)
Thank you,Mohab

**Comment by andrewtelnov** [comment]
@MohabElgammal I have added into our TODO list, but I have to say, we are not really enthusiastic about this functionality. I am not sure how to set an accessKey in the survey JSON model and UI for editing it in the SureyJS Editor.
At the current moment a markdown support in title and choices.text is in our radar. There are many requests that can be resolved by using a markdown.

Thank you,
Andrew

**Comment by MohabElgammal** [comment]
ok
Thank you for your efforts,
Mohab

**Comment by MohabElgammal** [comment]
Hello @andrewtelnov  
Is there any progress in Shortcuts ?

**Comment by andrewtelnov** [comment]
@MohabElgammal Are you talking about html "accesskey" attribute?

Thank you,
Andrew

**Comment by MohabElgammal** [comment]
@andrewtelnov :No, "accesskey" is varying in different browsers we need a way to specify default shortcuts in multiple choice or radio group button questions, like 1,2,3 or A,B,C for every question, the problem is that multiple questions could have the same shortcuts, did you check out typeform? it has this feature and it's very useful, it would be great to see it in surveyjs. 
Thank you

**Comment by andrewtelnov** [comment]
@MohabElgammal Could you please ping me in a month?
We are working on releasing version 1.0 and there are a lot of tasks in our TODO list to accomplish this goal.
I just feel this functionality can wait for awhile.

Thank you,
Andrew

**Comment by MohabElgammal** [comment]
Hey @andrewtelnov ,
I just have a suggestion regarding default shortcuts; you will find a library in the attachment. It has an html with a UX similar to typeform with a JavaScript library the separates each element in the form into its own which will allow (I think) multiple keyboard shortcuts (A, B, C …) to be applied to multiple choices or radio buttons on the same page, please run the demo and tell me what you think, and whether or not you will consider changing the UX of survey.js to be similar to typeform. Thanks.



[FullscreenForm.zip](https://github.com/surveyjs/editor/files/1292671/FullscreenForm.zip)

**Comment by JaneSjs** [comment]
Hi Everyone,
You can integrate a third-party library which offers the key entry functionality within SurveyJS Form Library. For instance, review the following ticket to get a demo using the hotkeys-js library: [Handling key events inside survey](https://surveyjs.answerdesk.io/internal/ticket/details/T11381).
